scala2.13/pom.xml

```diff
@@ -881,11 +880,14 @@
             331,
             332,
             333,
+            334,
             340,
             341,
+            342,
             350
         </noSnapshotScala213.buildvers>
         <snapshotScala213.buildvers>
+            351
```

Comment:
Actually, we should remove this too since it is removed from `snapshot.buildvers`

Reply:
Oh I forget it, let me update